go.mod

```diff
@@ -54,8 +54,8 @@ require (
 	go.uber.org/zap v1.11.0 // indirect
 	golang.org/x/sys v0.0.0-20191127021746-63cb32ae39b2
 	golang.org/x/time v0.0.0-20190921001708-c4c64cad1fd0 // indirect
-	k8s.io/api v0.17.0
-	k8s.io/apimachinery v0.17.0
+	k8s.io/api v0.17.13
```

Comment:
Please indicate in the commit message that you are bumping k8s to address this CVE.

Reply:
updated